core/variable_map.ts

```diff
@@ -420,6 +398,12 @@ export class VariableMap
    * @returns All of the variable names of all types.
    */
   getAllVariableNames(): string[] {
+    deprecation.warn(
+      'VariableMap.getAllVariableNames',
+      'v12',
+      'v13',
+      'Blockly.Variables.getAllVariables',
+    );
     const names: string[] = [];
```

Comment:
Can you have this call the method on the `Variables` namespace? or does that cause circular dependencies?

Reply:
~yes, but since this is deprecated and that would require iterating the list twice I'd just assume leave it alone.